src/main/datatypes/MQ2GroupType.cpp

```diff
@@ -97,6 +135,8 @@ bool MQ2GroupType::GetMember(MQVarPtr VarPtr, const char* Member, char* Index, M
 		return false;
 	}
 
+	float furthest_dist = 0;
```

Comment:
This should be in the scope of the members that need it, there's no reason to declare it out here.